client/src/Components/Alerts/AlertList.js

```diff
@@ -1,15 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import { Redirect, useHistory} from 'react-router-dom';
+import { Redirect, useHistory } from 'react-router-dom';
 
 import webHost from '../../config/web';
 import api from '../../config/web';
 import CreateAlert from './CreateAlert';
+import { PROFILE } from "../../constants/constants";
```

Comment:
in a previous pull request, I had mentioned this `PROFILE` constant should be defined either in a constant called something like `LOCAL_STORAGE_KEYS` or in it's own constants file

Reply:
can you set that as part of this pull request. It's not very clear what PROFILE is supposed to be just by reading the code

Reply:
```
const LOCAL_STORAGE_KEYS = {
   PROFILE: 'profile'
}
```

Reply:
`import { PROFILE } from "../../constants/localStorageKeys";`